irohad/model/execution/command_executor.hpp

```diff
@@ -246,7 +246,22 @@ namespace iroha {
       bool isValid(const Command &command,
                    ametsuchi::WsvQuery &queries) override;
     };
-  
+
+    class SetAccountDetailExecutor : public CommandExecutor {
+     public:
+      SetAccountDetailExecutor();
+
+      bool execute(const Command &command, ametsuchi::WsvQuery &queries,
+                   ametsuchi::WsvCommand &commands) override;
```

Comment:
This method (in cpp) doesn't modify (and even use) the object so make it const

Reply:
same to the protected ones

Reply:
Execute change state of bd, I am not sure, that this is a good idea.

Reply:
According to the code it doesn't perform any action on/with the object (so it's even can be static, though not applicable due to override)